src/components/Book.js

```diff
@@ -5,21 +5,47 @@ import '../styles/Book.css';
 const Book = ({
   title, author, category, onDelete,
 }) => (
-  <div>
-    <h3>{title}</h3>
-    <p>{author}</p>
-    <p>{category}</p>
-    <button type="button" onClick={onDelete}>
-      Delete
-    </button>
+  <div className="book">
+    <div className="book-content">
+      <div className="book-info">
+        <h4 className="book-category">{category}</h4>
+        <h2 className="book-title">{title}</h2>
+        <h6 className="book-author">{author}</h6>
+        <div className="action-buttons">
+          <button className="button-outline" type="button">Comments</button>
+          <div className="vertical-divider" />
+          <button className="button-outline" type="button" onClick={onDelete}>
+            Delete
+          </button>
+          <div className="vertical-divider" />
+          <button className="button-outline" type="button">Edit</button>
+        </div>
+      </div>
+      <div className="progress-container">
```

Comment:
Good job working on the project requirements but I noticed you did not follow the exact design rules for this project. You need to adjust a few things like using correct font-family, font size , font color etc. ill attach screenshots with a list of corrections you need to make.

Here is your design:


![Screenshot 1](https://github.com/ClarenceAtim/bookstore/assets/79506105/3426b1b0-2e1e-4f1a-a31c-6f87915e24cb)


Please stick to this design as much as possible:


![Screenshot 2023-07-22 at 2 44 53 PM](https://github.com/ClarenceAtim/bookstore/assets/79506105/c13d1c33-0623-4b1e-b0e7-cafb732a5890)


It would be nice to pay attention to design rules to avoid making mistakes in future.